src/app/components/planner-board/planner-board.component.less

```diff
@@ -97,25 +97,25 @@
   }
   .dropdown {
     a {
-      color: $color-pf-dark;
+      color: @color-pf-dark;
```

Comment:
it looks like `@color-pf-black-800`. Just remove the `@color-pf-dark` variable from `shared/_custom-variables.less`